src/highlighting.js

```diff
@@ -98,13 +99,13 @@ export default class Highlighting {
   // Manage Highlights
   // -----------------
 
-  editableHasChanged (editableHost, throttle) {
+  editableHasChanged (editableHost, throttle, raiseEvents) {
```

Comment:
Why do you pass `raiseEvents` here?

Don't we want to trigger the spellcheck event every the spellcheck highlight has updated?

Reply:
I didn't trigger it on the update of the spellcheck out of the framework. But this is not necessary anymore. So it makes more sense to always trigger it. I will remove it.